debian/copyright

```diff
@@ -0,0 +1,23 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: gst-perf
+Upstream-Contact: support@ridgerun.com
+
+License: LGPL-2+
```

Comment:
Should it be RidgeRun LGPL v2 license version or generic LGPL v2 for debian packages?

```
Copyright (C) <year> RidgeRun, LLC (http://www.ridgerun.com)
 
 This program is free software; you can redistribute it and/or modify
 it under the terms of the GNU General Public License version 2 as
 published by the Free Software Foundation.
```